src/views/Profile.vue

```diff
@@ -1,90 +1,104 @@
+      <v-btn
+        label="House Code"
+        class="mt-3 black--text"
+        @click="copyText"
+        color="orange lighten-2"
+        block
+        rounded
+      >
+        Copy House Key</v-btn
+      >
+    </v-card>
 
-       <v-divider></v-divider>
-      
-      <v-card class="text-center orange lighten-4 pa-3 mb-3" elevation="0">
-       <v-list-item-subtitle class="font-weight-bold mt-4  d-flex justify-center title ">
-          Invite people to your home!
-        </v-list-item-subtitle>
-        <v-btn
-            label="House Code"
-            class="mt-3 black--text"
-            @click="copyText"
-            color="orange lighten-2"
-            block
-            rounded
-          > Copy House Key</v-btn>
-      
-      </v-card>
-       
-         <v-divider></v-divider>
+    <v-divider></v-divider>
 
     <v-card class="d-flex justify-center orange lighten-4" elevation="0">
-        <v-list-item-subtitle class="font-weight-bold mt-4  d-flex justify-center">
-          Personal Info
-        </v-list-item-subtitle>
+      <v-list-item-subtitle
+        class="font-weight-bold mt-4  d-flex justify-center"
+      >
+        Personal Info
+      </v-list-item-subtitle>
     </v-card>
-    
 
     <v-list-item two-line>
       <v-list-item-content>
         <v-list-item-subtitle class="font-weight-bold mb-2 text--primary">
           Username
         </v-list-item-subtitle>
-        <v-list-item-subtitle class="font-weight-normal pl-4 text--primary">{{this.username}}</v-list-item-subtitle>
+        <v-list-item-subtitle class="font-weight-normal pl-4 text--primary">{{
+          this.username
+        }}</v-list-item-subtitle>
       </v-list-item-content>
     </v-list-item>
 
     <v-list-item two-line>
       <v-list-item-content>
-
         <v-list-item-subtitle class=" font-weight-bold mb-2 text--primary">
           Email
         </v-list-item-subtitle>
-        <v-list-item-subtitle class="font-weight-normal pl-4 text--primary">{{this.email}}</v-list-item-subtitle>
-
+        <v-list-item-subtitle class="font-weight-normal pl-4 text--primary">{{
+          this.email
+        }}</v-list-item-subtitle>
       </v-list-item-content>
     </v-list-item>
-    
-    <v-card class="mt-2 mb-2 d-flex justify-space-between orange lighten-4" elevation="0">
 
+    <v-card
+      class="mt-2 mb-2 d-flex justify-space-between orange lighten-4"
+      elevation="0"
+    >
       <UpdateUser @setProfilePageUsername="changeUsername" />
       <UpdatePhoto />
-
     </v-card>
-
-
-
+    <!-- this is the mailto button, we'll want to move this deeper when we aren't all worried about getting jobs😉 -->
+    <v-card
+      class="mt-2 mb-2 d-flex justify-center orange lighten-4"
+      elevation="0"
+    >
+      <a href="mailto:fruityraptors@gmail.com" target="_blank" class="email-button"
+        ><v-btn color="green lighten-2 black--text mb-5">
+          Email the dev team
+        </v-btn></a
+      >
+    </v-card>
```

Comment:
Super nitpicky... but

1. proper indentation
2. maybe added `rounded` to v-btn so it matches the rest

Reply:
will do this now...